Analytics/Integrations/Kahuna/SEGKahunaIntegration.m

```diff
@@ -80,17 +62,22 @@ - (void)start
         // We just need one call to launchWithKey and not multiple. The "start" method is called
         // everytime the app comes to foreground.
         if ([SEGKahunaPushMonitor sharedInstance].kahunaInitialized == NO) {
-            [KahunaAnalytics launchWithKey:apiKey];
-
+            [self.kahunaClass setDebugMode:true];
```

Comment:
Just removed that line setDebugMode. It was added in by mistake. Segment iOS has a debug more setting, but there is no easy way to get it. The variable itself is a static variable in a implementation class that we cannot add in our wrapper layer.